### PR TITLE
Add documentation for update expressions.

### PR DESCRIPTION
**Pull request by jpinner-lyft**

### CODE REVIEW

File: docs/updates.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+PynamoDB supports creating update expressions from attributes using a mix of built-in operators and method calls.
+Any value provided will be serialized using the serializer defined for that attribute.
+
+.. csv-table::
+    :header: DynamoDB Action / Operator, PynamoDB Syntax, Example
+
+    SET, set( `value` ), Thread.views.set(10)
+    REMOVE, remove(), Thread.subjects.remove()
+    ADD, add( `value` ), "Thread.subjects.add({'A New Subject', 'Another New Subject'})"
```
Comment (jmphilli) [inline docs/updates.rst]: this quoted third column -- on purpose?

File: docs/updates.rst [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    DELETE, delete( `value` ), Thread.subjects.delete({'An Old Subject'})
+    `attr_or_value_1` \+ `attr_or_value_2`, `attr_or_value_1` \+ `attr_or_value_2`, Thread.views + 5
+    `attr_or_value_1` \- `attr_or_value_2`, `attr_or_value_1` \- `attr_or_value_2`, 5 - Thread.views
+    "list_append( `attr` , `value` )", append( `value` ), Thread.notes.append(['my last note'])
+    "list_append( `value` , `attr` )", prepend( `value` ), Thread.notes.prepend(['my first note'])
+    "if_not_exists( `attr`, `value` )", `attr` | `value`, Thread.forum_name | 'Default Forum Name'
```
Comment (jmphilli) [inline docs/updates.rst]: can you add some examples to https://github.com/pynamodb/PynamoDB/tree/master/examples

File: docs/updates.rst (hunk 1 above)
Comment (jpinner-lyft) [reply docs/updates.rst]: you have to add quotes whenever you have a comma

File: docs/updates.rst (hunk 2 above)
Comment (jpinner-lyft) [reply docs/updates.rst]: updated model examples

**Pull request merged**